releases/release-1.22/release-notes/major-themes.md

```diff
@@ -0,0 +1,59 @@
+
+[A new Alpha feature](https://github.com/kubernetes/kubeadm/issues/2473) for running the kubeadm control plane components as non-root users. This is a long requested security measure in kubeadm. To try it you must enable the kubeadm-specific `RootlessControlPlane` feature gate. After we improved the sleep for security folks, they can drink their morning cup of beverage a bit less hastily.
+
+### Lets `kubectl debug` with the new and improved API for Ephermal Containers
+
+The API used to create Ephemeral Containers changed in 1.22. The Ephemeral Containers feature is alpha anddisabled by default, and the new API will not work with clients attempting to use the old API.
+Users who create ephemeral containers using kubectl debug should note that kubectl version 1.22 will attempt to fallback to the old API, but older versions of kubectl will not work with cluster versions of 1.22 or later. Please update kubectl to 1.22 if you wish to use kubectl debug with a mix of cluster versions. (This only applies to debugging using ephemeral containers.)
+Admission plugin authors should note that the AdmissionReview request will now contain the entire v1.Pod rather than just a list of ephemeral containers as before.
+Cluster administrators should note that securityContext may now be set on ephemeral containers and ensure that cluster security policies are applied to these fields prior to enabling the Ephemeral Containers feature.
+
+### Server-side Apply graduates to GA in 1.22.
+
+Server-side Apply is a new object merge algorithm, as well as tracking of field ownership, running on the Kubernetes api-server. It helps users and controllers manage their resources via declarative configurations. It allows them to create and/or modify their objects declaratively, simply by sending their fully specified intent. After being in beta for a couple releases, Server-side Apply graduates to GA in 1.22.
+
+### SIG Storage Moves couple features to GA
+
+CSI Windows moves to GA in the 1.22 release. Windows currently does not support privileged containers. To solve this problem, [CSIProxy](https://github.com/kubernetes-csi/csi-proxy) enables CSI node plugins to bedeployed as unprivileged pods and use the proxy to perform privileged storage operations on the node.
+CSI Service Account Token moves to GA in 1.22. This feature allows CSI drivers to use pods' [bound service account tokens](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md) instead of a more privileged identity. It also provides a knob to re-publish volumes so that short-lived volumes can be refreshed. 
```

Comment:
```suggestion
### Cluster Storage Interface improvements

SI support for Windows nodes moves to GA in the 1.22 release. Windows currently does not support privileged containers. To solve this problem, [CSIProxy](https://github.com/kubernetes-csi/csi-proxy) enables CSI node plugins to be deployed as unprivileged pods and use the proxy to perform privileged storage operations on the node.

Another feature moving to GA in v1.22 is CSI Service Account Token support. This feature allows CSI drivers to use pods' [bound service account tokens](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/#bound-service-account-token-volume) instead of a more privileged identity. It also provides control over to re-publishing these volumes, so that short-lived tokens can be refreshed. 
```